waku/v2/node/rest/debug/debug_api.nim

```diff
@@ -33,8 +33,8 @@ proc installDebugApiHandlers*(router: var RestRouter, node: WakuNode) =
 
 #### Client
 
-proc decodeBytes*(t: typedesc[DebugWakuInfo], data: openArray[byte], contentType: string): RestResult[DebugWakuInfo] =
-  if MediaType.init(contentType) != MIMETYPE_JSON:
+proc decodeBytes*(t: typedesc[DebugWakuInfo], data: openArray[byte], contentType: Opt[ContentTypeData]): RestResult[DebugWakuInfo] =
```

Comment:
To be sure, the proc signature was changed in the submodule and this simply reflects that in order to compile. That said, I imagine it was for convenience sake to use a direct alias for `Result` that indicates more of a binary outcome (rather than multiple error conditions), but still being able to use the `Result` API for which patterns have been established.

Reply:
It is a `nim-presto` thing. It used to be an option, the only thing is that it was not annotated as an option. Nim libraries' things...